src/components/Dialog.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Rnd from 'react-rnd';
+import { connect } from 'react-redux';
+import { toggleDialog } from '../ducks/dialog';
+import { Utility } from '../lib/Utility';
+
+class Dialog extends React.Component {
+  constructor(props) {
+    super(props);
+    this.popupBody = null;
+    this.close = this.close.bind(this);
+  }
+
+  onClose() {
+    this.props.dispatch(toggleDialog(null));
+  }
+
+  close(e) {
+    this.onClose();
+    return Utility.stopEvent(e);
+  }
+
+  render() {
+    const height = this.props.size.height;
+    const width = this.props.size.width;
+    const x = (window.innerWidth / 2) - (width / 2);
+    const y = ((window.innerHeight / 2) - (height / 2)) + window.pageYOffset;
+
+    const defaultPosition = { x, y, height, width };
+
+    return (
+      <Rnd
+        default={defaultPosition}
+        enableResizing={false}
+        minHeight={400}
+        minWidth={400}
+      >
+        <div
+          className="popup dialog"
+          tabIndex="0"
+          ref={(c) => { this.popupBody = c; }}
+          role="button"
+          onClick={(e) => { return e.target === this.popupBody && this.close(e); }}
```

Comment:
WAIT! I forgot one thing!

Does this ever trigger? I think it was a carry over from the other screen-blocking Popup modal, where if you clicked the semi-transparent overlay, it was the equivalent of the Close action. In fact, I'm not sure we even need `this.popupBody`

Reply:
Gotcha. I can remove this in a future PR.